bin/sandbox

```diff
@@ -99,7 +99,11 @@ unbundled bin/rails generate solidus:install \
   --auto-accept \
   $@
 
+unbundled bundle add lookbook listen actioncable --group development,test
```

Comment:
For some reason, it looks like the gems are not added to the sandbox's `Gemfile`.

Reply:
Is it a silent error or you get anything more specific? 
Also I suggest checking if it's the latest version of bundler, just to be sure.

Reply:
Very silent :slightly_smiling_face: Do you get it added in your sandbox?

Reply:
It is now part of the install generator, does it work for you now?

Reply:
I tried it yesterday and it was there.

Reply:
I tried now and it works, but I left a comment about the approach.